code/modules/mob/living/carbon/human/species_types/skeletons.dm

```diff
@@ -17,4 +17,4 @@
 /datum/species/skeleton/check_roundstart_eligible()
 	if(SSevents.holidays && SSevents.holidays[HALLOWEEN])
 		return TRUE
-	return ..()
\ No newline at end of file
+	return ..()

```

Comment:
hmm

Reply:
This is fine, it's just adding the EoF newline